content/forgero-vanilla/src/main/resources/data/forgero/packs/minecraft-material/type/material/hard_material.json

```diff
@@ -12,7 +12,7 @@
       "name": "HARD_PLANT"
     },
     {
-      "name": "PEARL"
+      "name": "WOOD"
```

Comment:
Was this a mistake though? Or did you just try adding it to more places?

Reply:
I fixed it.

Reply:
ah yes this was me trying to test if adding more the same types breaks it. Forgot to remove I guess mb